src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -803,4 +821,35 @@ public function hasAutowiringType($type)
 
         return isset($this->autowiringTypes[$type]);
     }
+
+    /**
+     * Returns all changes tracked for the Definition object.
+     *
+     * @return array An array of changes for this Definition
+     */
+    public function getChanges()
+    {
+        return $this->changes;
+    }
+
+    /**
+     * Turn internal change tracking on or off.
+     *
+     * @param bool $trackChanges
+     *
+     * @return $this
+     */
+    public function setTrackChanges($trackChanges)
+    {
+        $this->trackChanges = $trackChanges;
+
+        return $this;
+    }
+
+    private function trackChange($type)
```

Comment:
could be called "touch" (first read did tell me what it did - ie could have been the setter also, with boolean arg)